### ISSUE TITLE
How to use Server-Side Row Model for ag-Grid

### ISSUE THREAD

**Issue by maxkoretskyi** [widgetti/ipyaggrid#46]
[This example](https://gitlab.com/DGothrek/ipyaggrid/-/blob/master/docs/notebooks/demo-build-dataframe-output-html.ipynb) of how to use ipyaggrid shows the initialization of the agGrid with a fully loaded data (frame), which corresponds to the [client-side row model](https://www.ag-grid.com/javascript-data-grid/client-side-model/).

```
`g = Grid(**grid_data = data**, grid_options=grid_options, export_mode="buttons", theme='ag-theme-balham')`
``

Does ipyaggrid support agGrid in the [Server-Side Row Model](https://www.ag-grid.com/javascript-data-grid/server-side-model-datasource/)? If so, where can I read about it? If not, could you give me some guidelines on what I should do to enable it?

**Comment by mariobuikhuizen** [comment]
You're correct that ipyaggrid currently doesn't support agGrid's Server-Side Row Model. It's an area that we think would benefit the package, and we appreciate your interest in contributing.

Given the potential complexities of implementing this feature, we think that creating a new widget specifically for this functionality would be the most prudent approach. This would allow us to maintain compatibility for users who are already using the existing Grid widget.

While our team may not have the bandwidth for extensive guidance, we're definitely open to receiving a PR.

**Comment by maartenbreddels** [comment]
To clarify, the new widget could live in the ipyaggrid package as a new widget, it does not need to be a new ipywidget library/package.
